wiki/Shortcut_Key_Reference/ko.md

```diff
@@ -1,56 +1,282 @@
+- `Ctrl` + `Alt` + `Mouse Wheel Up`: 배치/선택 모드 전환 (위쪽으로)
+- `Ctrl` + `Alt` + `Mouse Wheel Down`: 배치/선택 모드 전환 (아래쪽으로)
+- `Ctrl` + `Shift` + `A` : [AiMod](/wiki/Beatmap_Editor/AiMod) 창 열기
+
+#### 플레이필드 설정
+
+- `Alt`: 키를 누르는 동안 [Distance Snap](/wiki/Beatmap_Editor/Distance_Snap) 토글
+- `Y`: Distance Snap 켜기/끄기
+- `Shift`: 키를 누르는 동안 Grid Snap 토글
+- `T`: Grid Snap 켜기/끄기
+- `G`: 격자 간격 변경
+- `Ctrl` + `1`: 격자 간격을 Tiny로 변경
+- `Ctrl` + `2`: 격자 간격을 Small로 변경
+- `Ctrl` + `3`: 격자 간격을 Medium으로 변경
+- `Ctrl` + `4`: 격자 간격을 Large로 변경
+- `L`: Lock Notes 켜기/끄기
+
+#### 오브젝트 조작
+
+- `Q`: New Combo 켜기/끄기
+- `Ctrl` + `,`: 반시계 방향으로 90도 회전
+- `Ctrl` + `.`: 시계 방향으로 90도 회전
+- `Ctrl` + `H`: 가로로 뒤집기
+- `Ctrl` + `J`: 세로로 뒤집기
+- `Ctrl` + `G`: 선택 반전
+- `Ctrl` + `Shift` + `R`: 선택한 노트의 회전 조절 (새 창으로 열림)
+- `Ctrl` + `Shift` + `S`: 선택한 노트의 거리 조절 (새 창으로 열림)
+- `Ctrl` + `Shift` + `D`: 다각형 모양으로 배치된 노트 만들기 (새 창으로 열림)
+- `Ctrl` + `Shift` + `F`: 슬라이더를 스트림으로 변환 (새 창으로 열림)
+- `Ctrl` + `Left Arrow`: 선택한 노트를 왼쪽으로 이동
+- `Ctrl` + `Right Arrow`: 선택한 노트를 오른쪽으로 이동
+- `Ctrl` + `Left Arrow`, `Right Arrow` (osu!mania 전용): 선택한 노트를 column 단위로 좌우로 이동
+- `Ctrl` + `Up Arrow`: 선택한 노트를 위쪽으로 이동
+- `Ctrl` + `Down Arrow`: 선택한 노트를 아래쪽으로 이동
+- `Ctrl` + `Up Arrow`, `Down Arrow` (osu!mania 전용): 선택한 노트를 tick 단위로 상하로 이동
+
+#### 히트사운드
+
+- `W`: 선택한 오브젝트에 Whistle 추가
+- `E`: 선택한 오브젝트에 Finish 추가
+- `R`: 선택한 오브젝트에 Clap 추가
+- `Shift` + `Q`, `W`, `E`, `R`: 선택한 오브젝트의 Sampleset 변경 (Auto, Normal, Soft, Drum)
+- `Ctrl` + `Q`, `W`, `E`, `R`: 선택한 오브젝트의 Additions 변경 (Auto, Normal, Soft, Drum)
+- `Ctrl` + `Shift` + `I`: 샘플 가져오기 창 열기
+
+#### 타임라인
+
+- `Shift` + `1`, `2`, `3`, `4`, `6`, `8`: [비트 배치 나눗수](/wiki/Beatmap_Editor/Beat_Snap_Divisor) 변경 (1/1, 1/2, 1/3, 1/4, 1/6, 1/8)
+- `Shift` 누르면서 서클 이동 (타임라인에서): 시간 그리드를 무시하고 배치 (osu!mania에서는 작동하지 않음)
+- `J`: 서클을 앞쪽 시간으로 이동
+- `K`: 서클을 뒤쪽 시간으로 이동
+- `L`: Lock Notes 켜기/끄기
+
+### Design
+
+- `I`: Sprite library 창 열기
+- `Ctrl` + `Z`: 실행 취소
+- `Ctrl` + `Y`: 다시 실행
+- `Ctrl` + `X`: 선택한 레이어 잘라내기
+- `Ctrl` + `C`: 선택한 레이어 복사하기
+- `Ctrl` + `V`: 현재 시간에 레이어 붙여넣기 (Note: 붙여넣기한 레이어는 맵을 저장하거나 다시 불러올 때까지 우측의 레이어 리스트에 나타나지 않음)
+- `Delete`: 선택한 레이어 삭제
+- `W`, `A`, `S`, `D`: 선택한 레이어를 각각 상하좌우로 1px만큼 이동
+
+### Timing
+
+- `T`: 키를 눌러서 [BPM](/wiki/BPM)과 offset 설정
+  - 일정하게 키를 눌러야 정확한 값을 얻을 수 있습니다.
+- `Shift` 누른 채로 BPM/offset 조절: 4배로 증가/감소
+- `Ctrl` + `B`: 현재 타이밍 포인트 초기화
+- `Ctrl` + `P`: 현재 시간에 타이밍 포인트 추가 (timing 화면이 아니라면 Timing Setup 창이 열림)
+- `Ctrl` + `Shift` + `P`: 현재 시간에 상속된 타이밍 포인트 추가 (Timing Setup 창이 열림)
+- `Ctrl` + `I`: 현재 타이밍 구역 삭제 (첫 번째 타이밍 포인트는 삭제하지 않음)
+
+### 마우스 조작
+
+- `Left Click`: 히트 서클 배치, 슬라이더 시작점 또는 배치 중인 슬라이더에 포인트를 추가, 서클 선택
+- `Ctrl` + `Left Click`: 기존 슬라이더에 새 포인트 추가
+- `Right Click`: New Combo 토글, 슬라이더 배치를 끝냄, 포인트를 제거, 오브젝트 삭제 (선택 모드에서만)
+- `Mouse Wheel Up/Down`: 앞/뒤 시간으로 1틱만큼 이동
+- `Shift` + `Mouse Wheel Up/Down`: 앞/뒤 시간으로 4틱만큼 이동
+- `Ctrl` + `Mouse Wheel Up/Down`: 비트 배치 나눗수 증가/감소
+- `Alt` + `Mouse Wheel Up/Down`: 플레이 영역에서는 거리 간격 조절, 타임라인 영역에서는 확대/축소
```

Comment:
when you change what the content says compared to English, update the English article as well. that way we can review the changes and keep all translations consistent with each other

I'll update this one

Reply:
Right, this part changed when I merged PR #2867. Thanks for letting me know!